apps/web/src/utils/poolCandidate.ts

```diff
@@ -715,3 +716,19 @@ export const getCombinedStatusLabel = (
     statusLabelKey,
     `Invalid statusLabelKey '${statusLabelKey}'`,
   );
+
+export const getPriorityWeight = (priorityWeight: number): PriorityWeight => {
+  if (priorityWeight === 10) {
+    return PriorityWeight.PriorityEntitlement;
+  }
+
+  if (priorityWeight === 20) {
+    return PriorityWeight.Veteran;
+  }
+
+  if (priorityWeight === 30) {
+    return PriorityWeight.CitizenOrPermanentResident;
+  }
+
+  return PriorityWeight.Other;
+};
```

Comment:
Just to be clear your sugegstion of:

> Should we also update the user.priorityWeight field to be the PriorityWeight enum as well, and call that method when needed?

Would make this function not necessary, correct? Since when we query for `user.priorityWeight` we would get the enum back instead? :thinking: I like that since working with numbers directly is kind of confusing and the transformation should be in one spot to reduce error.

Reply:
Ya this wouldn't be needed anymore since we can get the `PriorityWeight` enum directly from the query. I'd rather make another PR since we want to run it by other devs first.